presto-main/src/main/java/io/prestosql/server/ServerConfig.java

```diff
@@ -16,14 +16,11 @@
 import io.airlift.configuration.Config;
 import io.airlift.units.Duration;
 
-import javax.validation.constraints.NotNull;
-
 import static java.util.concurrent.TimeUnit.MINUTES;
 
 public class ServerConfig
 {
     private boolean coordinator = true;
-    private String prestoVersion = getClass().getPackage().getImplementationVersion();
```

Comment:
Let's run this in a separate PR to get feedback (if any) on this.

Reply:
This is required for the next change. This config was added in ancient times for no good reason. I can get it merged first if you’re concerned.

Reply:
I extracted this to https://github.com/prestosql/presto/pull/3708